### PR TITLE
Add yacs recipe

### PR DESCRIPTION
**Pull request by hadim**
Needed for #6951.

### CODE REVIEW

**Comment by hadim** [comment]
The license file is not available in the pypi source package. Two options:

- we don't ship the license file.
- We use the GitHub repo as a source and so we can ship the license file.

You guys decide.

**Comment by hadim** [comment]
I am not sure if I am supposed to ping the core team here. I saw some PRs being merged recently. 

Would you mind reviewing this one@ocefpaf?

File: recipes/yacs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  license: Apache 2.0
+  license_family: Apache
+  summary: 'YACS -- Yet Another Configuration System'
+  description: YACS -- Yet Another Configuration System
+  doc_url: https://github.com/rbgirshick/yacs
+  dev_url: https://github.com/rbgirshick/yacs
+
+extra:
+  recipe-maintainers:
+    - rbgirshick
```
Comment (scopatz) [inline recipes/yacs/meta.yaml]: @rbgirshick, are you ok being a maintainer?

Comment (hadim) [reply recipes/yacs/meta.yaml]: @rgrinberg I am fine if you don't want. I'll remove you.

Comment (rbgirshick) [reply recipes/yacs/meta.yaml]: @hadim thanks for being the maintainer.

**Comment by hadim** [comment]
@scopatz Looks good to merge.

**Pull request merged**